client-ts/Microsoft.AspNetCore.SignalR.Test.Server/Startup.cs

```diff
@@ -18,6 +28,43 @@ public void ConfigureServices(IServiceCollection services)
                 // consistent casing makes it cleaner to verify results
                 options.JsonSerializerSettings.ContractResolver = new DefaultContractResolver();
             });
+
+            services.AddAuthorization(options =>
+            {
+                options.AddPolicy(JwtBearerDefaults.AuthenticationScheme, policy =>
```

Comment:
This policy isn't being used anywhere. The Authorize attribute is referencing the authentication handler, not the policy.